### PR TITLE
Move domains into its own file

### PR DESCRIPTION
**Pull request by Weredime**
It makes it much easier to view the domains now.
Also fixes a bug, since chrome requires us to use DNR instead of webRequestBlocking for MV3!

### CODE REVIEW

**Comment by Weredime** [comment]
cc @immattdavidson

File: manifest.json [hunk 1]
```diff
@@ -8,20 +8,17 @@
...
-        "service_worker": "blockpage.js"
+        "service_worker": "blockpage.js",
+        "type": "module"
   },
   "icons": {
       "16": "assets/icon16.png",
       "48": "assets/icon48.png",
       "128": "assets/icon128.png"
   },
   "permissions": [
```
Comment (immattdavison) [inline manifest.json:19]: Please provide some form of reasoning for removing other permissions

File: domains.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+export default [
```
Comment (immattdavison) [inline domains.js:1]: some registrars that are listed are missing here.

**Review by immattdavison — CHANGES_REQUESTED** [review]

File: manifest.json (hunk 1 above)
Comment (Weredime) [reply manifest.json:19]: We do not require those for the extension yet

File: domains.js (hunk 2 above)
Comment (Weredime) [reply domains.js:1]: Sorry! While I was working on this PR i guess you guys added new domains. Commit incoming :rocket:

File: manifest.json (hunk 1 above)
Comment (immattdavison) [reply manifest.json:19]: Ok will merge once others have been resolved

**Comment by immattdavison** [comment]
@Weredime reverted this PR due to errors 
image

**Comment by Weredime** [comment]
> @Weredime reverted this PR due to errors image

Could you please open that and show it to me? I cannot replicate.

**Comment by immattdavison** [comment]
image

@Weredime

**Comment by Weredime** [comment]
@immattdavison I've tried to prevent that error in #17 - please test

**Pull request merged**